src/object/srv_obj.c

```diff
@@ -2079,6 +2083,8 @@ obj_ioc_init(uuid_t pool_uuid, uuid_t coh_uuid, uuid_t cont_uuid, crt_rpc_t *rpc
 
 	D_ASSERT(ioc != NULL);
 	memset(ioc, 0, sizeof(*ioc));
+
+	crt_req_addref(rpc);
```

Comment:
What is the additional RPC reference used for?

Reply:
@gnailzenh added this part

Reply:
@gnailzenh ping:)

Reply:
it would be good to clarify this part before this gets merged...